### ISSUE TITLE
TEAM.md -> EDITORS.md?

### ISSUE THREAD

**Issue by annevk** [whatwg/meta#64]
If we change this, and I think we should given the conflict with GitHub teams, it needs to be changed in whatwg/meta and whatwg/html. Anwhere else?

(We could also go for MAINTAINERS.md potentially, given that it's not always the editor that merges.)

**Comment by domfarolino** [comment]
FWIW I like MAINTAINERS.md, for the reason you gave at the end there.

**Comment by annevk** [comment]
Thanks @snuggs for taking care of this!

**Comment by snuggs** [comment]
@annevk oh crap. Probably should have not used Fixes in this case. Perhaps Addresses.

My bust. A convention i've used in OSS for dependencies when you have a lot to maintain.

Can you re-open? 

Dank je ;-)

https://help.github.com/articles/closing-issues-using-keywords/

**Comment by snuggs** [comment]
oh ok as long as #65  is still open. Whew. ;-)

**Comment by annevk** [comment]
@snuggs yeah, I think it's fine that this one is closed. I fiddled a bit with your commit messages to make sure nothing else got closed. 😊

**Comment by snuggs** [comment]
@annevk #TeamWorkToMakeTheDreamWork 🎉
